src/coreclr/inc/gcinfodecoder.h

```diff
@@ -565,15 +559,8 @@ class GcInfoDecoder
     UINT32  m_InstructionOffset;
 
     // Pre-decoded information
+    GcInfoHeaderFlags m_headerFlags;
     bool    m_IsInterruptible;
-    bool    m_IsVarArg;
-    bool    m_GenericSecretParamIsMD;
-    bool    m_GenericSecretParamIsMT;
```

Comment:
These are derived from masking the header flags. Masking is cheap and we may not even need these, so we can do masking in the accessor.